src/backend/distributed/planner/multi_master_planner.c

```diff
@@ -355,6 +384,39 @@ BuildAggregatePlan(Query *masterQuery, Plan *subPlan)
 		}
 	}
 
+	SS_attach_initplans(root, subPlan);
+
+#if PG_VERSION_NUM >= 110000
+
+	/*
+	 * If any Params were generated, run through the plan tree and compute
+	 * each plan node's extParam/allParam sets.  Ideally we'd merge this into
+	 * set_plan_references' tree traversal, but for now it has to be separate
+	 * because we need to visit subplans before not after main plan.
```

Comment:
"before not after main" -> "before, not after, main"

Reply:
I copied this piece verbatim from PG source code. I'll add a comment to indicate that. Should I still add the commas in that case?

Reply:
Also github has a nice feature for comments like this (first markup button on the left with the +-):
```suggestion
	 * because we need to visit subplans before, not after, main plan.
```

With that you can just merge little things like that from the github interface.